### ISSUE TITLE
Default schema filter doesn't filters failed_jobs table

### ISSUE THREAD

**Issue by driesvints** [laravel-doctrine/migrations#38]
For some reason, the default filter doesn't seems to work for me. Whenever I run `php artisan doctrine:migrations:diff` it creates a migration that tries to drop the `failed_jobs` table. Any idea what might be wrong?

```php
'schema' => [
    'filter' => '/^(?!password_resets|failed_jobs).*$/',
],
```

**Comment by ChrisSchwerdt** [comment]
I'm experiencing the same issue. It appears to be a bug in the `ConfigurationFactory` class. I've submitted #39 to fix this issue.

As a temporary workaround you can specify your schema filter on the command line using `php artisan doctrine:migrations:diff --filter-expression='/^(?!password_resets|failed_jobs).*$/'`, though I'd recommend adding an anchor to the list of table names for more specific matching: `/^(?!(password_reset_tokens|failed_jobs)$).*$/`.

**Comment by patrickbrouwers** [comment]
Released as 1.1.1
